QRCodeHere/ContentView.swift

```diff
@@ -88,6 +92,25 @@ struct ContentView: View {
         .onAppear(perform: updateQRContent)
     }
     
+    var watermarkView: some View {
+        Section {
+            if !addWatermark {
+                Button("Add watermark") {
+                    addWatermark.toggle()
+                }
+            }
+            if addWatermark {
```

Comment:
I think `else` does the job better here.
```suggestion
            } else {
```

Reply:
We can also `withAnimation` later on